source/uwp/Renderer/lib/AdaptiveToggleInput.h

```diff
@@ -25,6 +25,9 @@ namespace AdaptiveNamespace
         IFACEMETHODIMP get_Title(_Outptr_ HSTRING* title);
         IFACEMETHODIMP put_Title(_In_ HSTRING title);
 
+        IFACEMETHODIMP get_Wrap(_Outptr_ boolean* wrap);
+        IFACEMETHODIMP put_Wrap(_In_ boolean wrap);
```

Comment:
>_In_ [](start = 32, length = 4)

Remove unneeded annotation on data type

Reply:
when annotation is necessary?

---
In reply to: [245450181](https://github.com/Microsoft/AdaptiveCards/pull/2264#discussion_r245450181) [](ancestors = 245450181)

Reply:
The short answer is when it provides information that we can't otherwise encode in the language, such as whether it's expected that the caller will be passing in a value for a pointer (_In_) or if it's expected to be set as an out parameter (_Out_). In the case of a plain data type, the annotation doesn't tell us anything (a plain data type is already definitionally "In"). 

---
In reply to: [245450686](https://github.com/Microsoft/AdaptiveCards/pull/2264#discussion_r245450686) [](ancestors = 245450686,245450181)